pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -206,6 +207,12 @@ func (m *ManagerImpl) Start(activePods ActivePodsFunc, sourcesReady config.Sourc
 
 	socketPath := filepath.Join(m.socketdir, m.socketname)
 	os.MkdirAll(m.socketdir, 0755)
+	if selinux.SELinuxEnabled() {
+		err = selinux.SetFileLabel(m.socketdir, config.KubeletPluginsDirSELinuxLabel)
```

Comment:
This should really be `if err := x; err != nil`.  Using this style means that a later change in the code could access err, which is not what we want.